tests/profile/admin/test_profile_highlights_admin.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+from django.contrib.admin.sites import AdminSite
+from django.test import SimpleTestCase, TestCase
+
+from tests.profile.factories import ProfileHighlightFactory, ProfileFactory
+from tests.datasets.factories import DatasetFactory, IndicatorFactory
+from wazimap_ng.profile.models import ProfileHighlight
+from wazimap_ng.profile.admin.admins import ProfileHighlightAdmin
+
+class MockRequest:
+    pass
+
+
+class MockSuperUser:
+    def has_perm(self, perm, obj=None):
+        return True
+
+    @property
+    def is_superuser(self):
+        return True
+
+
+request = MockRequest()
+request.user = MockSuperUser()
+
+class ProfileHighlightAdminTests(TestCase):
+
+    @classmethod
+    def setUpTestData(self):
+        profile = ProfileFactory()
+        self.profile_highlights = ProfileHighlightFactory(profile=profile)
+        # qualitative data
+        dataset2 = DatasetFactory(profile=profile, content_type="qualitative")
+        self.qualitative_indicator = IndicatorFactory(dataset=dataset2)
+
+    def setUp(self):
+        self.site = AdminSite()
+
+    def test_modeladmin_str(self):
+        ma = ProfileHighlightAdmin(ProfileHighlight, self.site)
+        self.assertEqual(str(ma), 'profile.ProfileHighlightAdmin')
+
+
+    def test_indicator_queryset(self):
```

Comment:
I'm not 100% sure what you're testing? Are you checking that a profile highlight is not being created?

Reply:
perhaps change the method name to be more descriptive, e.g. test_that_qualitative_indicators_are_not_created